pkg/util/ovs/fake_ovs.go

```diff
@@ -0,0 +1,447 @@
+package ovs
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// ovsFake implements a fake ovs.Interface for testing purposes
+type ovsFake struct {
+	bridge string
+
+	ports map[string]int
+	flows []ovsFlow
+}
+
+// ovsFlow represents an OVS flow
+type ovsFlow struct {
+	table    int
+	priority int
+	created  time.Time
+	cookie   string
+	fields   []ovsField
+	actions  string
+}
+
+type ovsField struct {
+	name  string
+	value string
+}
+
+const (
+	minPriority     = 0
+	defaultPriority = 32768
+	maxPriority     = 65535
+)
+
+// NewFake returns a new ovs.Interface
+func NewFake(bridge string) Interface {
+	return &ovsFake{bridge: bridge}
+}
+
+func (fake *ovsFake) AddBridge(properties ...string) error {
```

Comment:
I was expecting AddBridge to take bridge name otherwise the function name is misleading. May be we want to call SetupBridge()